x-pack/plugins/security_solution/public/overview/components/entity_analytics/risk_score/columns.tsx

```diff
@@ -138,45 +138,13 @@ export const getRiskScoreColumns = (
     truncateText: false,
     mobileOptions: { show: true },
     render: (alertCount: number, risk) => (
-      <AlertsCountColumn
```

Comment:
I rolled back this change to deal with all side effects inside openEntityOnAlertsPage